[NativeAot] Document DirectPInvoke's requirements

At first glance, i thought i can add any DllImport'ed library.
But then i added `"Ultralight"` and few more.
I received a lot of errors.
Beginning of errors
```
Создается библиотека bin\Release\net5.0\win-x64\native\UltralightNet.Veldrid.TestApp.lib и объект bin\Release\net5.0\win-x64\native\UltralightNet.Veldrid.TestApp.exp
UltralightNet.Veldrid.TestApp.obj : error LNK2001: неразрешенный внешний символ ulApplyProjection. [C:\Users\Дом\source\repos\SupinePandora43\UltralightSharp\UltralightNet.Veldrid.TestApp\UltralightNet.Veldrid.TestApp.csproj]
UltralightNet.Veldrid.TestApp.obj : error LNK2001: неразрешенный внешний символ ulCreateRenderer. [C:\Users\Дом\source\repos\SupinePandora43\UltralightSharp\UltralightNet.Veldrid.TestApp\UltralightNet.Veldrid.TestApp.csproj]
UltralightNet.Veldrid.TestApp.obj : error LNK2001: неразрешенный внешний символ ulDestroyRenderer. 
```

Sounds like it requires `lib` files.
I have them!
But where they should be?
How do i specify where It should search for them?!

Documentation doesn't answer these questions.
I tried looking at corert repo, but didn't found.
Could documentation be improved?

~~i tried asking this on discord (DotNetEvolution) but it wasn't answered~~

--- a/docs/using-nativeaot/interop.md
+++ b/docs/using-nativeaot/interop.md
@@ -36,7 +36,7 @@ Examples:
 
 ### Linking
 
-To statically link against a native library, you'll need to specify `<NativeLibrary>filename</NativeLibrary>` pointing to `.lib` file
+To statically link against a unmanaged library, you'll need to specify `<NativeLibrary>filename</NativeLibrary>` pointing to `.lib` file on Windows and `.a` file on Unix
 
 Examples:
 
@@ -44,8 +44,9 @@ Examples:
 <ItemGroup>
   <!-- Generate direct PInvokes for Dependency -->
   <DirectPInvoke Include="Dependency" />
-  <!-- Specify .lib file -->
-  <NativeLibrary Include="Dependency.lib" />
+  <!-- Specify library to link against -->
+  <NativeLibrary Include="Dependency.lib" Condition="$(RuntimeIdentifier.StartsWith('win'))" />
+  <NativeLibrary Include="Dependency.a" Condition="!$(RuntimeIdentifier.StartsWith('win'))" />
 </ItemGroup>
 ```
 

--- a/docs/using-nativeaot/interop.md
+++ b/docs/using-nativeaot/interop.md
@@ -36,7 +36,7 @@ Examples:
 
 ### Linking
 
-To statically link against a unmanaged library, you'll need to specify `<NativeLibrary>filename</NativeLibrary>` pointing to `.lib` file on Windows and `.a` file on Unix
+To statically link against a unmanaged library, you'll need to specify `<NativeLibrary Include="filename" />` pointing to `.lib` file on Windows and `.a` file on Unix
 
 Examples:
 

--- a/docs/using-nativeaot/interop.md
+++ b/docs/using-nativeaot/interop.md
@@ -34,6 +34,21 @@ Examples:
 </ItemGroup>
 ```
 
+### Linking
+
+To statically link against a native library, you'll need to specify `<NativeLibrary>filename</NativeLibrary>` pointing to `.lib` file
+
+Examples:
+
+```xml
+<ItemGroup>
+  <!-- Generate direct PInvokes for Dependency -->
+  <DirectPInvoke Include="Dependency" />
+  <!-- Specify .lib file -->
+  <NativeLibrary Include="Dependency.lib" />
+</ItemGroup>
+```
+
 ## Native Exports
 
 The native AOT compiler will export methods annotated with `UnmanagedCallersOnlyAttribute` and explicitly specified name as

